src/resolvers/user.js

```diff
@@ -181,13 +181,19 @@ module.exports = {
       async (
         _,
         { id, updateUserInput },
-        { models: { User }, secret },
+        { models: { User }, currentUser, secret },
       ) => {
         try {
           const { userErrors, valid } = validateUpdateUserInput(
             updateUserInput,
           );
-
+          if (currentUser.id !== id && currentUser.role !== 'ADMIN') {
```

Comment:
@manny-e1 why do we need to check if the user has 'ADMIN' role to update his profile?

Reply:
@manny-e1 sry my bad